app/src/test/java/com/x8bit/bitwarden/data/platform/datasource/disk/ConfigDiskSourceTest.kt

```diff
@@ -106,6 +106,6 @@ private val SERVER_CONFIG = ServerConfig(
             notificationsUrl = "http://localhost:61840",
             ssoUrl = "http://localhost:51822",
         ),
-        featureStates = mapOf("duo-redirect" to true, "flexible-collections-v-1" to false),
+        featureStates = mapOf("duo-redirect" to "true", "flexible-collections-v-1" to "false"),
```

Comment:
Looks like the sample JSON above returns `true`, not `"true"`? Does the API return a string or boolean?

Reply:
ConfigDiskSource will the values as strings in order for them to be casted into the 3 possible types.